Backend/src/v1/data.py

```diff
@@ -14,6 +13,7 @@
     tags=["Data"],
 )
 
+
 @router.get("/", status_code=200)
 def get_all_nodes(collection: str) -> list[Node]:
```

Comment:
get_all_nodes() results in 500 response.. see the below errors:

```sh
backend-1  | pydantic_core._pydantic_core.ValidationError: 3 validation errors for Node                                                                                                                                                                      
backend-1  | id
backend-1  |   Field required [type=missing, input_value={'text': 'Hur utvecklades...el': 2, 'fine label': 4}, input_type=dict]                                                                                                                              
backend-1  | x                                                                                                                                                                                                                                               
backend-1  |   Field required [type=missing, input_value={'text': 'Hur utvecklades...el': 2, 'fine label': 4}, input_type=dict]                                                                                                                              
backend-1  | y                                                                                                                                                                                                                                               
backend-1  |   Field required [type=missing, input_value={'text': 'Hur utvecklades...el': 2, 'fine label': 4}, input_type=dict]
```

Reply:
This is strange to me tbh, i have checked the db inside docker after a first-run adn the collections look as they should. Maybe just a validator update is needed.

Reply:
This should be fixed now

Reply:
tested!!